integration_tests/suite/test_calls_recording.py

```diff
@@ -106,6 +105,28 @@ def event_received():
             not_(raises(CalldError))
         )
 
+    def test_put_record_start_from_user_errors(self):
+        user_uuid = str(uuid.uuid4())
+        token = self.given_user_token(user_uuid)
+        self.calld_client.set_token(token)
+        other_channel_id = self.given_call_not_stasis()
+        assert_that(
+            calling(self.calld_client.calls.start_record_from_user).with_args(UNKNOWN_UUID),
+            raises(CalldError).matching(has_properties(status_code=404))
+        )
+        assert_that(
+            calling(self.calld_client.calls.start_record_from_user).with_args(other_channel_id),
+            raises(CalldError).matching(has_properties(status_code=403))
+        )
+
+        variables = {'XIVO_CALLRECORDFILE': ''}
+        channel_id = self.given_call_not_stasis(variables=variables)
+
+        assert_that(
+            calling(self.calld_client.calls.start_record_from_user).with_args(channel_id),
+            raises(CalldError).matching(has_properties(status_code=403))
+        )
+
```

Comment:
Zuul is dead, this probably does not pass tests

Reply:
Well it passes tests but I don't know why. Isn't the status_code supposed to be 500 in this case?